PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -217,7 +217,12 @@ def miniAOD_customizeCommon(process):
                   'RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_50ns_V1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_25ns_nonTrig_V2p1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_50ns_nonTrig_V2p1_cff']
-    switchOnVIDPhotonIdProducer(process,DataFormat.MiniAOD) 
+    switchOnVIDPhotonIdProducer(process,DataFormat.AOD) 
```

Comment:
@ikrav 

I have a naive question.

miniAOD_tools is for reading the objects from miniAOD data format.

Why do we have to switch the data format to AOD which will read out "gedGsfElectrons" not in miniAOD?

Regards,
Taejeong